XCTestBootstrapTests/FBTestManagerTestReporterJUnitTests.m

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <OCMock/OCMock.h>
+#import <XCTest/XCTest.h>
+
+#import "FBTestManagerAPIMediator.h"
+#import "FBTestManagerResultSummary.h"
+#import "FBTestManagerTestReporterJUnit.h"
+
+#define testSuiteDidStart(name, startTime)                                                                             \
```

Comment:
Is there a good reason for using Macros here? You could define each of these as instance methods of the test case and get the same effect. Macros will make this harder to debug when a test fails since all of the parameters for the assertions will be inlined.